docs/tutorial.rst

```diff
@@ -618,6 +618,102 @@ Once you are satisfied that the demographic history that you have built
 is correct, it can then be simulated by calling the :func:`.simulate`
 function.
 
+-------------
+Census events
+-------------
+
+Census events allow you to add a node to each branch of the tree sequence at a given time during the simulation. This can be useful when you wish to study haplotypes that are ancestral to your simulated sample.
+
+For instance, the following code specifies a simulation with two samples drawn from each of two populations. There are two demographic events: a migration rate change and a census event. At generation 100 and earlier, the two populations exchange migrants at a rate of 0.05. At generation 5000, a census is performed.
+
+.. code-block:: python
+
+    >>> pop_config = msprime.PopulationConfiguration(sample_size=2, initial_size=1000)
+    >>> mig_rate_change = msprime.MigrationRateChange(time=100, rate=0.05)
+    >>> ts = msprime.simulate(
+                population_configurations=[pop_config, pop_config],
+                length=1000,
+                demographic_events=[mig_rate_change, msprime.CensusEvent(time=5000)],
+                recombination_rate=1e-7,
+                random_seed=141)
+
+The resulting tree sequence has nodes on each tree at the specified census time. These are the nodes with IDs 9, 10, 11, 12 and 13:
+
+.. code-block:: python
```

Comment:
We're trying to phase out using text trees in the documentation (#590) because they dont render properly in many browsers. Maybe we could do this with ``ts.draw_svg()``, and save the svg file (should be other examples of this, perhaps see the tskit tutorial for an example?)